### PR TITLE
Simplification of README

### PR DESCRIPTION
**Pull request by plbenveniste**
This pull request's objective is to simplify the use of the code. The main `README.md` only describes how to use our trained model. To learn more about how to retrain a similar model, refer to the `utils/README_full_process.md`.

The main works concern : 
- update of the main README.md file
- update of requirements.txt file with library versions
- creation of the README_full_process.md file
- fixed small changes in other files

### CODE REVIEW

**Review by jcohenadad — CHANGES_REQUESTED** [review]
There is a lot of technical details [here](https://github.com/ivadomed/model_seg_mouse-sc_wm-gm_t1/pull/42/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L32-L41) that are not appropriate for an introductory statement

File: README.md [hunk 1]
```diff
@@ -39,124 +24,39 @@ In order to train a 3D nnU-Net, the following steps were completed:
...
-~~~
-
-You will get the configuration plan for all four configurations (2d, 3d_fullres, 3d_lowres, 3d_cascade_fullres).
-
-> **Note**
-> In the case of the zurich_mouse dataset, nifti files are not fully annotated, therefore we use a 2d configuration.
-
-
-## Train model
+## Installation
```
Comment (jcohenadad) [inline README.md:29]: Consider changing for 'getting started'

File: README.md [hunk 2]
```diff
@@ -39,124 +24,39 @@ In order to train a 3D nnU-Net, the following steps were completed:
...
 
-To train the model, use the following command:
+Clone the repository:
```
Comment (jcohenadad) [inline README.md:31]:
```suggestion
This section explains how to install and use the model on new images. 

Clone the repository:
```

File: README.md (hunk 1 above)
Comment (jcohenadad) [inline README.md:29]:
```suggestion
If you wish to try the model on your own data, follow the instructions at #installation and #perform-predictions.

## Installation
```

Comment (jcohenadad) [reply README.md:29]: make sure my markdown hyperlinks work

File: README.md [hunk 3]
```diff
@@ -39,124 +24,39 @@ In order to train a 3D nnU-Net, the following steps were completed:
...
+## Perform predictions
 
-### Method 3 - Using our trained model with `test.py`
+To run an inference and obtain a segmentation, we advise using the following method (refer to `utils/README_full_process.md` for alternatives). 
 
 Download the model `Dataset500_zurich_mouse.zip` from the release and unzip it. 
 Use the `test.py` function:
 
-To run on the whole dataset:
+To run on an entire dataset:
```
Comment (jcohenadad) [inline README.md:59]: that is less intuitive-- i would start with 'single image'

File: README.md [hunk 4]
```diff
@@ -39,124 +24,39 @@ In order to train a 3D nnU-Net, the following steps were completed:
...
 ~~~
-CUDA_VISIBLE_DEVICES=XXX nnUNetv2_predict -i /path/to/image/folder -o /path/to/predictions -d 500 -c 3d_fullres --save_probabilities -chk checkpoint_best.pth -f 4
+pip install -r requirements.txt
 ~~~
 
-You can now access the predictions in the folder `/path/to/predictions`. 
+## Perform predictions
 
-### Method 3 - Using our trained model with `test.py`
+To run an inference and obtain a segmentation, we advise using the following method (refer to `utils/README_full_process.md` for alternatives).
```
Comment (jcohenadad) [inline README.md:54]: use hyperlinks

File: README.md [hunk 5]
```diff
@@ -39,124 +24,39 @@ In order to train a 3D nnU-Net, the following steps were completed:
...
+pip install -r requirements.txt
 ~~~
 
-You can now access the predictions in the folder `/path/to/predictions`. 
+## Perform predictions
 
-### Method 3 - Using our trained model with `test.py`
+To run an inference and obtain a segmentation, we advise using the following method (refer to `utils/README_full_process.md` for alternatives). 
 
 Download the model `Dataset500_zurich_mouse.zip` from the release and unzip it.
```
Comment (jcohenadad) [inline README.md:56]: link to latest release

File: README.md [hunk 6]
```diff
@@ -39,124 +24,39 @@ In order to train a 3D nnU-Net, the following steps were completed:
...
 ~~~
 
-You can now access the predictions in the folder `/path/to/predictions`. 
+## Perform predictions
 
-### Method 3 - Using our trained model with `test.py`
+To run an inference and obtain a segmentation, we advise using the following method (refer to `utils/README_full_process.md` for alternatives). 
 
 Download the model `Dataset500_zurich_mouse.zip` from the release and unzip it. 
 Use the `test.py` function:
```
Comment (jcohenadad) [inline README.md:57]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -166,6 +66,12 @@ To run on individual(s) NIfTI image(s):
 python test.py --path-images /path/to/image1 /path/to/image2 --path-out /path/to/output --path-model /path/to/nnUNetTrainer__nnUNetPlans__3d_fullres
 ~~~
 
+> The `nnUNetTrainer__nnUNetPlans__3d_fullres` folder is inside the `Dataset500_zurich_mouse` folder.
+
+> To use GPU, add the flag `--use-gpu` in the previous command.
+
+> To use mirroring (test-time) augmentation, add flag `--use-mirroring`. NOTE: Inference takes a long time when this is enabled. Default: False.
+
```
Comment (jcohenadad) [inline README.md:74]: use [MD admonitions](https://github.com/orgs/community/discussions/16925)

File: utils/README_full_process.md [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+
+We activate the environment:
+~~~
+conda activate venv_nnunet
+~~~
+
+Then install the required libraries:
+~~~
+pip install -r requirements.txt
+~~~
```
Comment (jcohenadad) [inline utils/README_full_process.md:50]: avoid duplication please

**Review by jcohenadad — CHANGES_REQUESTED** [review]
i suggest 
- removing the duplicated info between README and README_full_process
- renaming README_full_process --> README_training_model
- add a link to README_training_model inside README (eg section "Train model")

**Comment by plbenveniste** [comment]
Modifications done ! Ready for last review and merge.

File: README.md [hunk 9]
```diff
@@ -39,143 +27,68 @@ In order to train a 3D nnU-Net, the following steps were completed:
...
-## Data
-
-Download dataset (internal git-annex at NeuroPoly): `zurich-mouse`
-
-
-## Data pre-processing
+</details>
 
-Create the following folders:
+This `README` file shows how to use the model which we trained to infer predictions. For information on how to retrain the same model, refer to this file [README_training_model.md](https://github.com/ivadomed/model_seg_mouse-sc_wm-gm_t1/blob/main/utils/README_training_model.md).
```
Comment (jcohenadad) [inline README.md:32]:
```suggestion
For information on how to retrain the same model, refer to this file [README_training_model.md](https://github.com/ivadomed/model_seg_mouse-sc_wm-gm_t1/blob/main/utils/README_training_model.md). 
```

**Review by jcohenadad — APPROVED** [review]
Looks great!

**Pull request merged**